### DISCUSSION TITLE
Field for GPTQ-for-LLaMa

### DISCUSSION THREAD

**Discussion by flurb18** [oobabooga/text-generation-webui#1783] [Ideas]
It seems different quantized models need different repositories of GPTQ-for-LLaMa to work. Going forward maybe an idea is to have a dropdown menu in the model tab selecting which repository you're using?

**Comment by LaaZa** [comment]
Hopefully this will solve that whole thing #1668

**Reply by flurb18** [reply]
Interesting... I'm a fan of one size fits all solutions, but will it significantly impact performance? Also, will it be maintained for support for newer / less popular methods such as https://github.com/mayank31398/GPTQ-for-SantaCoder

**Reply by LaaZa** [reply]
The aim is to employ the best available optimisations in one package. Support for new model types can be added. You can go there and leave a request.

**Reply by eucdee** [reply]
> Interesting... I'm a fan of one size fits all solutions, but will it significantly impact performance? Also, will it be maintained for support for newer / less popular methods such as https://github.com/mayank31398/GPTQ-for-SantaCoder

oh nice is this a special GPTQ to make SantaCoder work better or work at all? lemme try it..
